### PR TITLE
feat(ci): Add configuration to run beta tests when a change is detected.

### LINKED ISSUE #11232: Trigger Desktop Enrollment and Full UI tests based on Desktop Beta builds
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4761)

### PR DESCRIPTION
**Pull request by b4handjr**
Because

- We want to run our Firefox Beta integration tests when we detect a new build upstream

This commit

- Adds the configurations for this to be done

Fixes #11232

### CODE REVIEW

File: .circleci/updater_script.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (jaredlockhart) [inline .circleci/updater_script.sh:1]: Shouldn't this go in `scripts/`?

Comment (b4handjr) [reply .circleci/updater_script.sh:1]: I can, I just felt it wasn't a script for general use, just mainly for CI.

Comment (jaredlockhart) [reply .circleci/updater_script.sh:1]: Totally, but it's more like, if I'm looking for a script, that's where I'd look.

Comment (b4handjr) [reply .circleci/updater_script.sh:1]: Makes sense, ill move it and rename it so it's a bit more specific

**Review by jaredlockhart — APPROVED** [review]
Awesome cool neat, let's go! 🎉 🎉 🎉

**Pull request merged**